### DISCUSSION TITLE
essay/2022/02/01/dont-use-template

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#186] [Comments]
# essay/2022/02/01/dont-use-template

C++로 코딩할 때 STL을 즐겨 사용했습니다. 대학교에서는 왜 이런 걸 가르쳐주지 않았을까? 학생 때 고생 고생해서 만든 동적 배열, 링크드 리스트, 맵 같은 것들이 여기 다 있었네. 내가 만든 것보다 훨씬 좋은 걸.

https://jeho.page/essay/2022/02/01/dont-use-template.html

**Comment by coketazo** [comment]
이 업계에 발을 들였다면 은퇴할때까지는 배우려는 자세를 유지해야한다고 생각합니다
